services/auth/source/oauth2/providers.go

```diff
@@ -59,7 +59,7 @@ func (p *AuthSourceProvider) DisplayName() string {
 
 func (p *AuthSourceProvider) IconHTML(size int) template.HTML {
 	if p.iconURL != "" {
-		img := fmt.Sprintf(`<img class="gt-object-contain gt-mr-3" width="%d" height="%d" src="%s" alt="%s">`,
+		img := fmt.Sprintf(`<img class="tw-object-contain gt-mr-3" width="%d" height="%d" src="%s" alt="%s">`,
```

Comment:
Does tailwind config scan "*.go" files?

This is the only use of "tw-object-contain"

Reply:
Ah we need to fix that.

Reply:
https://github.com/go-gitea/gitea/pull/29686